ddtrace/llmobs/_integrations/mcp.py

```diff
@@ -69,7 +98,17 @@ def _llmobs_set_tags_client(self, span: Span, args: List[Any], kwargs: Dict[str,
             }
         )
 
-        if span.error or response is None:
```

Comment:
is removing the span.error check intentional?

Reply:
yeah because we might have an error on the span that we manually added but it will still have an "output". i should add back in the check for if response is none tho i think

Reply:
oh actually i still have that check in here so yeah it's intentional we're not checking the error anymore